internal/datanode/compactor.go

```diff
@@ -97,9 +97,11 @@ func (t *compactionTask) mergeDeltalogs(dBlobs map[UniqueID][]*Blob, timetravelT
 	var (
 		pk2ts = make(map[UniqueID]Timestamp)
 		dbuff = &DelDataBuf{
-			delData: &DeleteData{Data: make(map[UniqueID]UniqueID)},
-			tsFrom:  math.MaxUint64,
-			tsTo:    0,
+			delData: &DeleteData{
+				Pks: make([]UniqueID, 0),
+				Tss: make([]Timestamp, 0)},
+			tsFrom: math.MaxUint64,
```

Comment:
in next pr, please change to use typeutil.MaxTimestamp and typeutil.ZeroTimestamp